src/app/models/route.ts

```diff
@@ -7,5 +7,6 @@ export interface Route {
   time: string;
   vehicleId: string;
   passengers: number;
+  distance: number;
```

Comment:
distance would be different for each option, so you would find it in RouteOption

Reply:
to be more clear - options would look like this:

```
{
  walking: {
     distance: 10.5,
     duration: 30.5
  },
  transit: {
     distance: 10.5,
     duration: 30.5
  },
  [...]
}
```